extension/src/controllers/subtitle-controller.ts

```diff
@@ -310,21 +312,25 @@ export default class SubtitleController {
                 this.onNextToShow?.(slice.nextToShow[0]);
             }
 
+            let shouldUnblur = false;
+            if (this.subtitleSettings) {
```

Comment:
This is relatively expensive code to be running ~10 times a second. I think there is a more performant approach.